app/javascript/src/components/editor/Compiler.svelte

```diff
@@ -41,14 +43,20 @@
 
 <svelte:window on:keydown={keydown} />
 
-<button class="button button--secondary button--square tooltip" on:click={doCompile}>
-  {#if compiling}
-    Compiling...
+<button
+  class={"tooltip " + ($$restProps["class"] ?? "") + " " + (inline ? "empty-button" : "button button--secondary button--square")}
```

Comment:
You can use `{ ... }` inside the property to make it a little more readable

```suggestion
  class="tooltip {$$restProps["class"]} {inline ? "empty-button" : "button button--secondary button--square"}"
```